grid-proxy/pkg/types/nodes.go

```diff
@@ -53,6 +55,11 @@ type CapacityResult struct {
 	Used  Capacity `json:"used_resources"`
 }
 
+type Speed struct {
```

Comment:
I noticed there're multiple speeds for upload / download there's ipv4, ipv6 versions? which is it? and are

Reply:
yes we have `ipv4` and `ipv6` for both `tcp` and `udp` these are four per an iperf server, 
for example devnet has two iperf servers on the two public nodes so we have total of `8` tests

a discussion was raised on the chat group and i remember the decision was to show the only non-zero value. should we do something different? or view them all?